frontend/src/static/js/v_zoom.js

```diff
@@ -29,9 +32,29 @@ window.vZoom = {
       filteredUser.commits = zUser.commits.filter(
           (commit) => commit[date] >= zSince && commit[date] <= zUntil,
       ).sort(this.sortingFunction);
+      this.selectedCommits = filteredUser.commits;
+      this.updateFileTypes(filteredUser.commits);
```

Comment:
We can make selectedFileTypes as a computed value, too

Reply:
Thanks for the review. Can I clarify why you think selectedFileTypes can be made a computed value? In my implementation, the updateFileTypes method is only called once, and filetypes can only be determined when the commits are known. You can think of it as a form of "initialization" of required variables. selectedFileTypes is only dependent on user interaction thereafter.

Reply:
Yes, I got what you mean. I didn't notice that this attribute will only dependent on user interaction after the initialization.